cmd/loki-canary/main.go

```diff
@@ -42,6 +42,10 @@ func main() {
 	queryTimeout := flag.Duration("query-timeout", 10*time.Second, "How long to wait for a query response from Loki")
 
 	interval := flag.Duration("interval", 1000*time.Millisecond, "Duration between log entries")
+	outOfOrderPercentage := flag.Int("out-of-order-percentage", 0, "Percentage (0-100) of log entries that should be sent out of order.")
+	outOfOrderMin := flag.Int("out-of-order-min", 30, "Minimum amount of time to go back for out of order entries (in seconds).")
```

Comment:
Let's make these `time.Duration` instead of an int measuring seconds for ease of use.

Reply:
thought this would make the math ugly later on, but it's fine since `<time.Duration>.Seconds()` exists, so done